.gitlab-ci.yml

```diff
@@ -346,6 +346,18 @@ TCID-DIR-DMAAS-SCHD-JIVA-MINIO-BUCKET:
     GROUP: dmaasJivaMinio
     THREADS: 5
 
+TCID-DIR-DMAAS-SCHD-JIVA-MYSQL-BUCKET:
```

Comment:
@bhaskarhc Use gui dmaas template for dmaas tests, Take reference from existing Dmaas job configuration.